tests/test_text.py

```diff
@@ -167,3 +167,75 @@ def test_text_zfill(data, fletcher_variant):
     # Pandas returns np.nan for NA values in cat, keep this in line
     result_fr[result_fr.isna()] = np.nan
     tm.assert_series_equal(result_fr, result_pd)
+
+
+@pytest.fixture(
+    params=[
+        [],
+        [""],
+        [None],
+        [" "],
+        ["\u2000"],
+        [" a"],
+        ["a "],
+        [" a "],
+        ["\u2000a\u2000"],
+        ["\u2000\u200C\u2000"],
+        ["\n\u200C\r"],
+        ["\u2000\x80\u2000"],
+        ["\t\x80\x0b"],
+        ["\u2000\u10FFFF\u2000"],
+        [" \u10FFFF "],
+    ]
+    + [
+        [c]
+        for c in " \t\r\n\x1f\x1e\x1d\x1c\x0c\x0b"
+        "\u2000\u2001\u2002\u2003\u2004\u2005\u2006\u2007\u2008\u2000\u2009\u200A\u200B\u2028\u2029\u202F"
+    ]
+    + [[chr(c)] for c in range(0x32)]
+    + [[chr(c)] for c in range(0x80, 0x85)]
+    + [[chr(c)] for c in range(0x200C, 0x2030)]
+    + [[chr(c)] for c in range(0x2060, 0x2070)]
+    + [[chr(c)] for c in range(0x10FFFE, 0x110000)],
+    scope="session",
+)
```

Comment:
You should be able to add this via the `@example` decorator to the `hypothesis`-based test. That should avoid having the test method twice.